art_common.hpp

```diff
@@ -36,6 +39,14 @@ using value_view = std::span<const std::byte>;
 /// length (std::span).
 using key_view = std::span<const std::byte>;
 
+/// A type alias determining the maximum size of a key that may be
```

Comment:
Most new comments in this file appear to be formatted with a too-narrow fill column